resource_obmcs_database_db_system.go

```diff
@@ -378,9 +409,12 @@ func (s *DBSystemResourceCrud) SetData() {
 	s.D.Set("backup_subnet_id", s.Res.BackupSubnetID)
 	s.D.Set("cluster_name", s.Res.ClusterName)
 	s.D.Set("data_storage_percentage", s.Res.DataStoragePercentage)
+	s.D.Set("data_storage_size_in_gb", s.Res.DataStorageSizeInGB)
```

Comment:
I don't see data_storage_size_in_gb in the schema. It looks like LaunchDbSystemDetails uses initialDataStorageSizeInGB, but DbSystem uses dataStorageSizeInGB. For the provider we should combine these into dataStorageSizeInGB.

Reply:
Excuse me, I'm still learning how to write a provider.  Is the schema shared between Resource and DataSource?  Anyway, it makes sense to combine and unify the attribute name.  I will still leave the data_storage_size_in_gb as ForceNew=true for now because we don't support UpdateDbSystem yet.